File: score.cpp
#include <stdio.h>

// return average
int find_avg (int* ptr, int N) {
    int avg = 0;
    for (int i=0;i<N;i++) {
        avg += *ptr;
        ptr++;
    }
    printf ("\navg: %d\n",avg/N);
    return avg/N;
}

//return count that counts the scores more than average
int count_max (int* ptr, int N, int avg) {
    int count =0;
    for (int i=0;i<N;i++) {
        if (*ptr>avg) {
            count++;
        }
        ptr++;
    }
    printf ("count:%d\n",count);
    return count;
}

int main (void) {
    int C;
    scanf ("%d",&C);
    printf ("C:%d\n",C);
    for (int i=0; i<C; i++) {
        int N;
        scanf ("%d",&N);
        printf ("N:%d\n",N);
         int score [N];
            for (int i=0;i<N;i++) {
                scanf ("%d",&score[i]);
            }
        printf ("score array: ");
            for (int j=0;j<N;j++) {
                 printf ("%d",score[j]);
                 printf (",");
            }
        int* ptr = &score[0];
        int avg = find_avg (ptr, N);
        int* temp = &score[0];
        int count = count_max (temp, N, avg);
        float ans = (float)count/N*100;
        printf ("%.3f%%\n",ans);
    }
    return 0;
}

File: python/score.py
국어 = int(input("국어 점수를 입력하세요:"))
수학 = int(input("수학 점수를 입력하세요:"))
영어 = int(input("영어 점수를 입력하세요:"))
avg = (국어+수학+영어)/3
print ("국어:",국어, "수학:",수학, "영어", 영어, "평균", avg)
if avg>60 :
    if (국어 or 수학 or 영어) <50:
        print("과락")
    else:
        print("합격")
else:
    print ("불합격")
